en/DatabaseProperties.md

```diff
@@ -1,28 +1,60 @@
 ---
 title: Database properties window
-outdated: true
 ---
 
 # Database properties window
 
-*Opened from the main window by selecting **File -&gt; Database properties***
+Each database can have specific properties that can be modified through **File -&gt; Database Properties**).
+These specific properties override the default properties (which are defined in **Options -&gt; Preferences**).
+
+The database-specific properties are stored in the database itself.
+This way, when moving the database to another computer, these properties are preserved. 
+In most of cases, special fields starting with *@Comment{jabref-meta:* are used. 
 
-The database properties window lets you set certain database-specific settings.
 
 ## Database encoding
 
 This setting determines which character encoding JabRef will use when writing this database to disk. Changing this setting will override the setting made in Preferences dialog for this database. JabRef specifies the encoding near the top of the bib file, in order to be able to use the correct encoding next time you open the file.
 
+The dropdown menu allows to select one encoding. UTF-8 is highly recommended.
+
+
 ## Override default file directories
 
-These settings are used to specify which directory will be used to search for general file links (specified in the *file* field), and for legacy PDF/PS links (the *pdf* and *ps* fields were used in JabRef versions prior to 2.3, but should in current versions be replaced by general file links).
+In your database, files (PDF, etc.) can be linked to an entry. The list of these files are stored in the *file* field of the entry. The location of these files have to be specified.
+
+For your database, you can define a **general file directory** and a **user-specific file directory**.
+These settings override the *main file directory* defined in the Preferences dialog.
 
 Relative directories can be specified. This means that the location of the files will be interpreted relative to the location of the bib file. Simply setting a directory to "." (without quotes) means that the files should reside in the same directory as the bib file.
 
-These settings override the general file directories specified in the Preferences dialog. If no values are set, the general directories will be used.
+*Note: the legacy PDF/PS links (i.e. the *pdf* and *ps* fields, which were used in JabRef versions prior to 2.3), should in current versions be replaced by general file links.*
+
```

Comment:
"This can be done using **Quality -&gt; Cleanup entries...** and selecting *Upgrade external PDF/PS links to use the 'file' field*."